packages/legacy/src/Plans/components/PlanActionsDropdown.tsx

```diff
@@ -148,41 +146,14 @@ export const PlanActionsDropdown: React.FunctionComponent<IPlansActionDropdownPr
                 </ConditionalTooltip>,
               ]
             : []),
-          <ConditionalTooltip
```

Comment:
How about adding this information to the delete dialog? i.e.  `Are you sure you want to delete Foo? This plan is currently running`

Reply:
we don't have a delete modal ... not sure adding a delete modal is in the scope here, but I can do it

Reply:
we have DeleteModal in the `planActions.sx`  - it should be possible to just update the text there

Reply:
there is a delete modal that is only for plans, it's not a re usable "delete modal" we can use for any resouces.

a. it use the deprecated  `ConfirmModal` so it will need to be replaced by a new component.
b. in there I already changed the text to be "Are you sure you want to delete Foo?"